packager/media/base/widevine_key_source_unittest.cc

```diff
@@ -514,7 +514,7 @@ const char kCryptoPeriodRequestMessageFormat[] =
     R"("tracks":[{"type":"SD"},{"type":"HD"},{"type":"UHD1"},)"
     R"({"type":"UHD2"},{"type":"AUDIO"}],)"
     R"("drm_types":["WIDEVINE"],)"
-    R"("first_crypto_period_index":%u,"crypto_period_count":%u,)"
+    R"("first_crypto_period_index":%u,"crypto_period_count":%u,"crypto_period_seconds":%u,)"
```

Comment:
Can you break it to two lines so the line length is less than 80 characters per line?

i.e.

R"("first_crypto_period_index":%u,"crypto_period_count":%u,)"
R"("crypto_period_seconds":%u,)"